### ISSUE TITLE
Targeting of specific limbs

### ISSUE THREAD

**Issue by rafaelkb** [CleverRaven/Cataclysm-DDA#41723]
**1** - The ability to shoot/hit a specific limb. When shooting **and** in melee, the player could press a specific key and aim for a limb they would get a chance for hitting it.
 Ex: head, arm, torso, etc.
 Hitting the legs the creature/npc would make them more slow.
 Hitting the arm would make the creature/npc drop his weapon.
Hitting the head could make a critical hit (see number 2 below)

This obviously would require the creatures and monsters HP get reworked but it would worth it.

**2** - The first thing is the "ability" to make the player and NPCs unconscious by hitting them in the head.
 If a player or NPC get hit with more than 60% of damage in the head they could be unconscious for some turns, even some hours in the case of NPCs.

Mi-gos could capture you when you get unconscious. 

 If number 2 gets added the ability to tie up monsters.

**Comment by Leland** [comment]
@rafaelkb I updated the issue title so this is a little more searchable in the future :)

**Comment by rafaelkb** [comment]
> @rafaelkb I updated the issue title so this is a little more searchable in the future :) 
> 
> Could you please separate out your 3rd item into its own issue? Big lists of disparate issues are really hard to "close"

Sure

**Comment by kevingranade** [comment]
One, do not make multiple suggestions in one issue.
Two, both of these suggestions have already been made and rejected multiple times.

Targeting body parts:
https://discourse.cataclysmdda.org/t/fms-frequently-made-suggestions/9764?u=kevin.granade
Tl;dr no, this leads to too much micromanagement and the link between hitting a body part and an outcome is not as straightforward as this implies. It would be a huge amount of implementation effort for something that would rarely be done, especially for shooting.

Hitting NPCs on the head to "knock them out"
This is a solution in search of a problem, there are pretty much zero reasonable use cases for knocking out a target.
Similarly  being captured by mi-go is game over, extending the scenario to some kind of prison escape is a non starter.
